config.yaml

```diff
@@ -0,0 +1,9 @@
+author: Deepak Sharma <deepshar@redhat.com>
+server: "https://f8a-analytics-preview-2445582058137.production.gw.apicast.io:443"
```

Comment:
Is it fine to expose this and the token?

Reply:
Actually, token is already exposed https://github.com/fabric8-analytics/fabric8-analytics-vscode-extension/blob/39f3e9ba63f609d0d93646032f23b60eb3a01fb1/src/apiendpoint.ts